### PR TITLE
Add yu.tf2pickup.eu

### PR DESCRIPTION
**Pull request by makemake-kbo**
Added tf2pickup.rs branding. Logo is a WIP and will be changed later.

### CODE REVIEW

File: src/environments/tf2pickup.rs.ts
Comment (garrappachc) [inline src/environments/tf2pickup.rs.ts:19]: Typo: Pickup-up

**Review by garrappachc — CHANGES_REQUESTED** [review]
The `tf2pickup_logo_white.png` file should be two times smaller (in each dimension) than `tf2pickup_logo_white@2x.png`. The `@2` version is picked up on high-density displays (4k monitors, mobile devices, etc.).

**Comment by makemake-kbo** [comment]
I updated the domain and fixed the typos. Im waiting for bobair to do the graphics so we should wait for her until we merge.
How about a version bump aswell?

**Comment by makemake-kbo** [comment]
I removed them now. Thanks for the heads up!

**Review by garrappachc — APPROVED** [review]

**Pull request merged**
